extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -64,6 +66,20 @@ export function RemoteQueries(): JSX.Element {
   const numOfReposToShow = repoListExpanded ? queryResult.results.length : numOfReposInContractedMode;
 
   try {
+    const openQueryFile = () => {
+      vscode.postMessage({
+        t: 'openFile',
+        filePath: queryResult.queryFilePath
+      });
+    };
+
+    const openQueryTextTmpFile = () => {
```

Comment:
I think this name was a leftover from the previous approach. Shall we call it `openQueryTextVirtualFile`?